src/corva/service/service.py

```diff
@@ -3,14 +3,19 @@
 
 from corva import shared
 from corva.service.api_sdk import ApiSdkProtocol
+from corva.service.cache_sdk import InternalCacheSdkProtocol
 
 
 def run_app(
     has_secrets: bool,
     app_key: str,
     api_sdk: ApiSdkProtocol,
+    cache_sdk: InternalCacheSdkProtocol,
     app: Callable[[], Any],
+    vacuum_delete_count: int = 3,
 ) -> Any:
+    cache_sdk.vacuum(delete_count=vacuum_delete_count)
```

Comment:
As we have custom `hset with expiry by key` implementation - expired keys are not deleted by redis automatically (only if whole hash expires). Vacuum scripts collects and deletes such expired hash keys and is called once an app invoke.